bitcoin/BRWalletManagerEvent.c

```diff
@@ -149,19 +149,27 @@ typedef struct {
     struct BREventRecord base;
     BRWalletManager manager;
     int rid;
-    BRTransaction *transaction;
+    uint8_t *transaction;
+    size_t transactionLength;
+    uint64_t timestamp;
+    uint64_t blockHeight;
```

Comment:
If the timestamp, blockHeight and length are so easily derived from the transaction, why must they be added to the event and flowed through all the event-y interfaces.

Reply:
Oh, because it is no longer a BRTransaction, it is just some bytes.

Reply:
Correct. Getting closer to a unified BTC/GEN client here...